### ISSUE TITLE
Encoder lights

### ISSUE THREAD

**Issue by SQLSammy** [maartentamboer/X-Touch-Mini-FS2020#51]
Hi,

can you add a simple option to the encoders to set light to current status (first, alternate1, alternate2),
like  "autolights": 0/1 and  "on"/"off"
which sets 1/2 or 3 lights depending on status

Andreas

**Comment by maartentamboer** [comment]
Yes, I will take this into consideration when I'll start to work on the encoder overhaul. I also have it in the planning to support configuring the encoder as a selector switch so it's no longer needed to create such a massive jinja function as the a320 range switch example.
